src/partisan_hyparview_peer_service_manager.erl

```diff
@@ -556,30 +600,45 @@ handle_message({neighbor_accepted, Peer, Tag, Sender},
     {reply, ok, State};
 
 handle_message({neighbor_rejected, Peer, _Sender}, State) ->
+    %% @todo Is this necessary?
```

Comment:
As I know, we did not add the peer to the active view before sending the `neighbor_request` message. I am not sure whether this trigger is needed.